apps/site/lib/site_web/templates/schedule/_line_page_stop_info.html.eex

```diff
@@ -21,8 +21,10 @@
 <div class="route-branch-stop-links">
   <%= if direction_id = schedule_link_direction_id(@stop, @reverse_direction_all_stops, @direction_id) do %>
     <% trip_path = trip_view_path(@conn, :show, @route.id, origin: @stop.id, direction_id: direction_id) %>
-    <%= link class: "route-branch-stop-link schedule-link", to: trip_path do %>
-      Schedules from here
+    <%= if @route.type != 1 do %>
```

Comment:
@ryan-mahoney it's not clear what this chunk has to do with removing the flag

Reply:
Eyes like a hawk!

Yeah, for some reason Ingrid requested that change as part of this ticket 🤷‍♂ 

![Screen Shot 2019-07-26 at 3 39 04 PM](https://user-images.githubusercontent.com/988609/61977111-86fd4080-afbb-11e9-8fd5-9ff9766ba2b9.png)